modules/storage-persistent-storage-pv.adoc

```diff
@@ -159,6 +160,7 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 //|GlusterFS  | ✅ | ✅ | ✅
 |HostPath  | ✅ | - |  -
 |IBM VPC Disk | ✅ | - |  -
+|{ibmpowerProductName} Virtual Server Disk | ✅ | ✅ |  ✅
```

Comment:
Same comment as above, I'd suggest moving this above "IBM VPC Disk" if there's no other reason for it to be in this particular order.